src/main/io/displayport_crsf.c

```diff
@@ -56,31 +54,27 @@ static int crsfClearScreen(displayPort_t *displayPort)
     return 0;
 }
 
-static int crsfRelease(displayPort_t *displayPort)
-{
+static int crsfRelease(displayPort_t *displayPort) {
     displayPort->grabCount = 0;
     return crsfClearScreen(displayPort);
 }
 
-static int crsfDrawScreen(displayPort_t *displayPort)
-{
+static int crsfDrawScreen(displayPort_t *displayPort) {
     UNUSED(displayPort);
     return 0;
 }
 
-static int crsfScreenSize(const displayPort_t *displayPort)
-{
+static int crsfScreenSize(const displayPort_t *displayPort) {
     return displayPort->rows * displayPort->cols;
 }
 
 
-static int crsfWriteString(displayPort_t *displayPort, uint8_t col, uint8_t row, const char *s)
-{
+static int crsfWriteString(displayPort_t *displayPort, uint8_t col, uint8_t row, const char *s) {
     UNUSED(displayPort);
     if (row >= crsfScreen.rows || col >= crsfScreen.cols) {
         return 0;
     }
-    const size_t truncLen = MIN((int)strlen(s), crsfScreen.cols-col);  // truncate at colCount
+    const size_t truncLen = MIN((int)strlen(s), crsfScreen.cols - col); // truncate at colCount
```

Comment:
Codacy found an issue: [Does not handle strings that are not \0-terminated; if given one it may perform an over-read (it could cause a crash if unprotected) (CWE-126).  ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6326343)